salt/preflight/mandatory.sls

```diff
@@ -0,0 +1,63 @@
+{%- from 'preflight/map.jinja' import settings with context %}
+
+install mandatory packages:
+  pkg.installed:
+    - pkgs: {{ settings.mandatory.packages }}
+
+{%- if settings.destructive %}
+{%- for directory in settings.mandatory.available_pathdirs %}
+make sure {{ directory }} does not exist:
+  file.absent:
+    - name: {{ directory }}
+{%- endfor %}
+{%- endif %}
+
+{%- for service in settings.mandatory.services.running %}
+enable {{ service  }} service:
+  service.running:
+    - name: {{ service  }}
+    - enable: true
+{%- endfor %}
+
+# required to set sysctl net.bridge.bridge-nf-call-iptables
+add the module br_netfilter to kernel:
+  kmod.present:
+    - name: br_netfilter
+    - persist: True
```

Comment:
Can we choose in which configuration file this should be stored, as we do for `sysctl` values, or is this not supported by the Salt state?